ui/src/frontend/notes_panel.ts

```diff
@@ -37,7 +37,7 @@ import {
 } from './gridline_helper';
 import {Panel, PanelSize} from './panel';
 import {isTraceLoaded} from './sidebar';
-import { customButtonRegistry } from './button_registry';
+import {customButtonRegistry} from './button_registry';
```

Comment:
Those deltas exist because we were "Saving without Formatting".  This meant we weren't using prettier at all.  As a result when Eslint and Prettier were added back in and configured correctly (https://github.com/android-graphics/sokatoa/pull/157) to use Perfetto's configurations that any lines edited by us or added before then, there will be some deltas to match Perfetto's formatting.